contrib/kube-redis-exporter-with-redis-sentinel.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: sentinel
+    role: service
+  name: redis-sentinel
+spec:
+  ports:
+    - port: 26379
+      targetPort: 26379
+  selector:
+    redis-sentinel: "true"
+---
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: redis
+spec:
+  serviceName: redis-sentinel
+  replicas: 1
+  selector:
+    matchLabels:
+      name: sentinel
+  template:
+    metadata:
+      labels:
+        name: sentinel
+        redis-sentinel: "true"
+#      annotations:
+#        security.alpha.kubernetes.io/unsafe-sysctls: net.core.somaxconn=1024
+    spec:
+      containers:
+      - name: redis
+        image: cookingcodewithme/redis:latest
```

Comment:
Why the `cookingcodewithme` image and not the official redis image?

Reply:
This is for testing of new docker image. I'll update the image path tonight.

Reply:
Ah ok. I'd prefer to wait till the redis image has the fix included, I'm a little hesitant to link to/endorse an image that's not the official one.